src/components/sidebar/side-menu.tsx

```
@@ -1,17 +1,224 @@
-import React from 'react';
-
-import { Box } from '@/components/ui/box';
-import { Text } from '@/components/ui/text';
+import { useRouter } from 'expo-router';
+import {
+  Contact,
+  FileText,
+  Home,
+  List,
+  type LucideIcon,
+  Map,
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)